web-stories.php

```diff
@@ -47,6 +47,7 @@
 define( 'WEBSTORIES_PLUGIN_DIR_URL', plugin_dir_url( WEBSTORIES_PLUGIN_FILE ) );
 define( 'WEBSTORIES_ASSETS_URL', WEBSTORIES_PLUGIN_DIR_URL . 'assets' );
 define( 'WEBSTORIES_MINIMUM_PHP_VERSION', '5.6.0' );
+define( 'WEBSTORIES_DEV_MODE', true );
```

Comment:
```suggestion
if( ! defined( 'WEBSTORIES_DEV_MODE' ) ) {
    define( 'WEBSTORIES_DEV_MODE', true );
}
```

This is an example where site owner might want to turn this on or off.

Reply:
Should this not default to off / false?
We could set it to true here https://github.com/google/web-stories-wp/blob/main/bin/local-env/install-wordpress.sh#L131-L135

Reply:
The idea is to turn this constant **off** for any production builds (still needs a ticket).

_For now_ I don't think this should be changeable by users as the screen is currently only useful for the development team. It's pretty useless for anyone else.

I like the idea originally laid out in #1738, but we are simply not there yet.

Why? Most flags are for things that ar not even implemented yet. For example, the "Keyboard Shortcuts" flag displays a button that does nothing. Right now, there's no point in giving end users a way to toggle this flag.

This is a somewhat unanswered question, see https://github.com/google/web-stories-wp/issues/1738#issuecomment-627400233

Reply:
I still think that the suggested change is valid.

Reply:
Is that the only thing blocking you from approving otherwise?

Reply:
It is the last thing and I will approve.